modules/core/comment/farm_comment.module

```diff
@@ -69,6 +71,49 @@ function farm_comment_base_field_definition(string $entity_type) {
   return $field;
 }
 
+/**
+ * Implements hook_entity_form_display_alter().
+ */
+function farm_comment_entity_form_display_alter(EntityFormDisplayInterface $form_display, array $context) {
+  if ($form_display->getTargetEntityTypeId() == 'comment' && $form_display->getMode() == 'default' && $form_display->isNew()) {
```

Comment:
I think we should only alter form/view displays for the `asset`, `log` and `plan` comment bundles we provide. I'll work on a change for this

Reply:
Done

Reply:
Oh, we also need to alter the `full` view mode which is used when viewing comments on the "Reply" page.